app/v1/dbControllers/user.js

```diff
@@ -0,0 +1,23 @@
+import pool from '../dbModel/connection';
+
+class UserController {
+	static signup(request, response) {
+		const selectQuery = {
+			text: 'INSERT INTO users (firstname, lastname, username, phoneNumber, email, password, isAdmin) VALUES($1, $2, $3, $4, $5, $6, $7) RETURNING *',
```

Comment:
Don't think you should be hardcoding data here. Try passing in the request body gotten from a Postman request instead

Reply:
It' s a query to insert into database.